### ISSUE TITLE
a11y: BigIcon relies on colour to convey destructive/success meaning

### ISSUE THREAD

**Issue by t3chguy** [element-hq/compound#343]
# Platform

- Web

# Description

https://compound.element.io/?path=/story/compound-web_icon-bigicon--destructive uses colour alone to convey the action is destructive/successful - this is not accessible

Ref https://element-io.atlassian.net/browse/PSB-968

<img width="1710" height="838" alt="Image" src="https://github.com/user-attachments/assets/3252ddc3-5b3a-4f01-b612-7939d3047344" />
